tests/reconfiguration.py

```diff
@@ -400,9 +400,7 @@ def test_join_straddling_primary_replacement(network, args):
                 "name": "transition_node_to_trusted",
                 "args": {
                     "node_id": new_node.node_id,
-                    "valid_from": str(
-                        infra.crypto.datetime_to_X509time(datetime.now())
-                    ),
+                    "valid_from": str(datetime.now()),
```

Comment:
This is awesome! As we now accept multiple time formats, could you update https://microsoft.github.io/CCF/main/governance/common_member_operations.html?highlight=asn1#renewing-node-certificate to mention this and add an entry to the changelog?